bip-XXXX.mediawiki

```diff
@@ -0,0 +1,160 @@
+<pre>
+  BIP: ?
+  Layer: Consensus (soft fork)
+  Title: Disallow 64-byte transactions
+  Author: Chris Stewart <stewart.chris1234@gmail.com>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-?
+  Status: Draft
+  Type: Standards Track
+  Created: ?
+  License: BSD-3-Clause
+</pre>
+
+==Abstract==
+
+This BIP describes the rationale for disallowing transactions that are serialized to 64 bytes without the transaction's witness. 
+We describe the weaknesses to the merkle tree included in bitcoin block headers, various exploits for those weaknesses.
+
+==Specification==
+
+This BIP disallows bitcoin transactions that are serialized to 64 bytes in length without its witness.
```

Comment:
s/its/their/, or "transaction" in the singular form

```suggestion
This BIP disallows bitcoin transactions that are serialized to 64 bytes in length without their witness.
```

Reply:
https://github.com/bitcoin/bips/pull/1760#discussion_r1987577059